modules/efk-logging-deploy-subscription.adoc

```diff
@@ -181,11 +182,11 @@ metadata:
 spec:
   channel: "preview" <2>
```

Comment:
The channel name is `"4.2"`, https://github.com/openshift/elasticsearch-operator/blob/release-4.2/manifests/elasticsearch-operator.package.yaml#L4